src/messenger-shortcuts.js

```diff
@@ -171,7 +171,7 @@ function sendLike() {
 }
 
 function searchInConversation() {
-  var targetNode = getByAttr('em', 'data-intl-translation', 'Search in Conversation');
+  var targetNode = document.querySelector('._3szn._3szo ._5odt') // return the first one that matches the selector
   click(targetNode);
```

Comment:
@sarangjo you are absolutely right.
@guoguo12 unfortunately I couldn't find a semantic selector, it seems that the rendered DOM for the controls in the info pane is now just a bunch of divs with classes and `role=button`